www/js/diary/LabelTab.tsx

```diff
@@ -214,23 +235,22 @@ const LabelTab = () => {
         fillLocationNamesOfTrip(trip);
       });
     const readTimelineMap = compositeTrips2TimelineMap(tripsRead, showPlaces);
+    logDebug(`LabelTab: after composite trips converted, 
+      readTimelineMap = ${JSON.stringify(readTimelineMap)}`);
     if (mode == 'append') {
       setTimelineMap(new Map([...timelineMap, ...readTimelineMap]));
     } else if (mode == 'prepend') {
       setTimelineMap(new Map([...readTimelineMap, ...timelineMap]));
     } else if (mode == 'replace') {
       setTimelineMap(readTimelineMap);
     } else {
-      return console.error('Unknown insertion mode ' + mode);
+      return displayErrorMsg('Unknown insertion mode ' + mode);
```

Comment:
@JGreenlee I'm moving this to the "cleanup" column after the merge so that we can revisit how the logger should display errors.

Reply:
`displayError` accepts an Error object as a parameter. `displayErrorMsg` is used when there is no error and we just want to popup with some text.

Maybe in cases where there is no error, we should be throwing one ourselves and then print out that stacktrace?

Reply:
> Maybe in cases where there is no error, we should be throwing one ourselves and then print out that stacktrace?

If we can do so easily, I think that would be helpful. Otherwise, we have to work backwards from the message to the code, and don't really have the full trace

Reply:
The stack traces are not very useful anyway because all the JS is bundled.

Maybe we could do something like https://stackoverflow.com/questions/58448000/how-to-get-a-readable-stack-trace-for-a-webpack-production-environment-for-a-min